client/components/LearnerViewDirective.js

```diff
@@ -71,6 +50,27 @@ tie.directive('learnerView', [function() {
               </div>
             </div>
           </div>
+          <div class="tie-question-ui">
+            <div class="tie-question-window">
+              <h3>Exercise {{exerciseNumber}}: {{title}}</h3>
```

Comment:
(Also, the alternative -- which, IIUC, entails putting "Exercise 1: Internationalization" as the title in the question JS file -- seems worse to me, since it implicitly breaks encapsulation.)